crates/aptos-rosetta/src/construction.rs

```diff
@@ -402,11 +402,11 @@ async fn construction_parse(
                 (AccountAddress::ONE, APTOS_ACCOUNT_MODULE, CREATE_ACCOUNT_FUNCTION) => {
                     parse_create_account_operation(sender, &type_args, &args)?
                 }
-                (AccountAddress::ONE, STAKE_MODULE, SET_OPERATOR_FUNCTION) => {
```

Comment:
I think we should still keep the old one and make this new proxy set operator a separate call. This allows most normal use cases to use the old one, which is simpler as it doesn't need to pass the old operator.

Reply:
I think this makes the Rosetta flow a little too complicated, you need to know if you're using X you need to store the operator.  if you're using Y you don't.